mods/other/real_suffocation/init.lua

```diff
@@ -0,0 +1,56 @@
+-- Load setting
+local suffocation_damage = 2
+local setting = minetest.settings:get("real_suffocation_damage")
```

Comment:
Better to add the mod name at the beginning.
```suggestion
local setting = minetest.settings:get("real_suffocation.real_suffocation_damage")
```

Reply:
I don't think this is really needed, @LoneWolfHT?

Reply:
e.g. turning the functions from local functions into table functions.

Reply:
I'm going to leave it as much Wuzzy's as I can, so although this is simple, we shouldn't need to override any of these functions (fix in source, instead) and thus using local functions is fine with me.
We use a lot of local functions in ctf, as example.